### ISSUE TITLE
Use swap line up/down to change task priorities

### ISSUE THREAD

**Issue by itamaro** [dertuxmalwieder/SublimeTodoTxt#12]
For example, if I'm on task marked as (A), and I swap it down below another task marked as (B), then it would be nice if it would auto-change the priority to (B) as well.

**Comment by dertuxmalwieder** [comment]
What if I just want to switch the lines but not the priorities?

**Comment by itamaro** [comment]
> What if I just want to switch the lines but not the priorities?

Good point. I guess this can be a syntax-specific optional feature, disabled by default.

**Comment by dertuxmalwieder** [comment]
Wouldn't that be quite an overkill for the limited use? :)

**Comment by itamaro** [comment]
I don't think so, but if you think it would, maybe it is :-)

**Comment by dertuxmalwieder** [comment]
Let me put it that way: If you send me a PR, I might consider to merge it.

**Comment by itamaro** [comment]
gotcha :-)

I'll look into it.
